resources/docker-compose-regtest-liquid.yml

```diff
@@ -166,6 +166,17 @@ services:
       local:
         ipv4_address: 10.10.0.17
     restart: unless-stopped
+  liquid-coach:
+    image: vulpemventures/liquid-coach:latest
+    container_name: liquid-coach
+    networks:
+      local:
+        ipv4_address: 10.10.0.18  
+    environment:
+      API_URL: ${LIQUID_LIQUID-COACH_URL}
```

Comment:
underscore

Reply:
also, what this does it mean? Is it for switching the Explorer backend of liquid coach?

Reply:
@francismars